iroh-one/src/rpc.rs

```diff
@@ -0,0 +1,25 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use iroh_rpc_types::gateway::{Gateway as RpcGateway, GatewayServerAddr, VersionResponse};
+
+#[derive(Default)]
+pub struct Gateway {}
+
+#[async_trait]
+impl RpcGateway for Gateway {
```

Comment:
Is this actually needed? given we also have a store and a p2p node wouldn't we proxy those too if we do this?

Reply:
Don't think it's used, carry over from @fabricedesre work.